### ISSUE TITLE
Crash on Recipe lookup

### ISSUE THREAD

**Issue by Gunner76th** [mezz/JustEnoughItems#760]
Thank you for reporting!

For issues:
 * If the issue seems to affect only one mod, report to them instead of here.
 * Remove NEI and FEI to see if they are causing the issue before reporting.
 * If you crashed, please paste the crash log to [gist](https://gist.github.com/) and link it here.

For suggestions:
 * Please do not suggest features from NEI. [See this spreadsheet](https://goo.gl/MQ0f3R) before requesting.
[crash-2017-03-07_17.02.14-client.txt](https://github.com/mezz/JustEnoughItems/files/824998/crash-2017-03-07_17.02.14-client.txt)
Clients are crashing when they do a lookup of the Furnace recipe for enderIO Conduit binder. NEI is NOT installed. 

Here is a copy of of the minetweaker scripts we are using also. We are running Sky Factory 3 by Bacon Donut. I do not know if they have anything to do with the crash, but figured it might help for you to have them to look at just in case. 
[scripts.zip](https://github.com/mezz/JustEnoughItems/files/825015/scripts.zip)

**Comment by mezz** [comment]
Thanks for the report! You should remove the issue template when you report.

This is caused by a mod creating an invalid Smelting Recipe. 
I pushed a fix that may keep JEI from crashing on it though. 
Please update JEI to build 416 when it's done building in a few minutes, and look in the client log for a line with this: "Found invalid smelting recipe: "

**Comment by drakray** [comment]
I got a similar crash, when pressing U on a SparkHammer Wooden Excavator
http://pastebin.com/3FZtfqkw

**Comment by mezz** [comment]
Thanks @drakray please report that as a new issue, it's not quite the same.

**Comment by Gunner76th** [comment]
As far as I know, we have not had any more issues with people crashing with just JEI installed. I also went through my logs from the 8th and 9th on my client, and did not see any of the output debug errors you mentioned to look for. I think that might be due to having fixed the furnace recipes for minetweaker when you identified what was causing it in your first reply. If you would like I can try to generate some output by intentionally enabling an invalid furnace recipe. All I did was comment out the invalid recipes in the script files with //.

**Comment by mezz** [comment]
Yes that would be helpful to confirm that the fix is working, thanks.

**Comment by Gunner76th** [comment]
Ok so I am not sure how to recreate the issue. When I tried to re-enable the refined storage furnace recipes that was causing it before, they dont load. Same as before, the mod was removed, but I forgot to comment out their references in the MT file. I just now re-enabled the recipes, did an /mt reload, and tried to look up the usages for the different items in the furnace recipe list. None of the recipes loaded, and when I look at the log file, it shows where MT wasnt even able to load the recipes due to invalid references to a mod that does not exist. 

If you have any thoughts on how to try and force the error, I am open to suggestions.
